website/cue/reference/configuration.cue

```diff
@@ -528,6 +528,36 @@ configuration: {
 						}
 					}
 				}
+				aws_secrets_manager: {
+					required: true
+					description: """
+						Retrieve secrets from AWS Secrets Manager.
+
+						The secret is expected to be a JSON text string with key/value pairs, e.g.:
+						```json
+						{
+                          "username": "test",
+                          "password": "example-password"
+                        }
+						```
+
+						Vector will log and exit if an error occurred retrieving the secrets.
+
+						Secrets will be loaded when Vector starts or if Vector receives a `SIGHUP` signal triggering its
```

Comment:
```suggestion
						Secrets are loaded when Vector starts or if Vector receives a `SIGHUP` signal triggering its
```

Reply:
i also updated to occurrence for the `exec` backend.